bankapp/script.js

```diff
@@ -0,0 +1,44 @@
+let action = "";
+let bal = 1000;
+let run = true;
+let amount = 0;
+let funds = 0;
+
+function banking() {
+    let wit = 0;
```

Comment:
You could do away with these two variable declarations and instead just assign them on the prompt

`let wit = prompt('give us a value')'`